tests/indexer/conftest.py

```diff
@@ -11,13 +11,22 @@
 
 @pytest.fixture()
 def make_wx_file(tmpdir):
-    def make(year, city, code, lon, lat, elev):
-        p = tmpdir.join(f"{year}s_CAN_BC_{city}.{code}_CWEC2016.epw")
-        p.write(
-            f"""LOCATION,{city},BC,CAN,CWEC2016,{code},{lat},{lon},-8.0,{elev} | Morphed:TAS,RHS,DWPT,PS | File Version: 2.1 | Creation Date: 2020-06-23
+    def make(year, city, code, lon, lat, elev, scenario = "RCP85", format = 1):
```

Comment:
I'll switch the name of the parameter to something like "backwards_compatible" since as far as I can tell we don't have any "format 1" files in the actual database, it's just maintained in the tests for backwards compatibility.

Reply:
hm, on second thought, I don't really know if we'll have format 1 files again in he future.